content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -19,6 +19,7 @@ A `datasource` block accepts the following fields:
 | `provider`          | **Yes**  | String (`postgresql`, `mysql`, `sqlite`, `sqlserver`, `mongodb`, `cockroachdb`) | Describes which data source connectors to use.                                                                                                                                                                                                                                                                                                    |
 | `url`               | **Yes**  | String (URL)                                                                    | Connection URL including authentication info. Most connectors use [the syntax provided by the database](/reference/database-reference/connection-urls#format).                                                                                                                                                                                    |
 | `shadowDatabaseUrl` | No       | String (URL)                                                                    | Connection URL to the shadow database used by Prisma Migrate. Allows you to use a cloud-hosted database as the shadow database.                                                                                                                                                                                                                   |
+| `directUrl`         | No       | String (URL)                                                                    | Connection URL for direct connection to the database.<br /><br />If you use the Data Proxy, the generated Prisma Client uses the URL defined by the `url` argument. All other connections use the URL defined by the `directUrl` argument.                                                                                                        |
```

Comment:
```suggestion
| `directUrl`         | No       | String (URL)                                                                    | URL for direct connection to the database.<br /><br />If you use the Data Proxy, the generated Prisma Client uses the URL defined by the `url` argument. All other connections use the URL defined by the `directUrl` argument.                                                                                                        |
```

Reply:
Perhaps we can skip "Connection" in all of these instances?

Reply:
Hm, "connection URL" is the [standard term we use](https://www.prisma.io/docs/reference/database-reference/connection-urls). Agree it's a bit repetitive though.